src/pages/video-test.tsx

```diff
@@ -78,7 +81,21 @@ const EggheadPlayer: React.FC<{videoResource: VideoResource}> = ({
                 kind="metadata"
                 label="notes"
               />
-              <CueBar key="cue-bar" order={6.0} />
+              <CueBar
+                onClickCue={() => {
+                  playerPrefs.setPlayerPrefs({sideBar: {activeTab: 0}})
```

Comment:
I think that making tabs data-driven might make sense now for cases like this. `0` won't always indicate Notes tab.

Reply:
give the tabs ids/names instead of generic indices